internal/handlers/publishers.go

```diff
@@ -19,62 +17,56 @@ func GetPublishers(ctx *fiber.Ctx) error {
 }
 
 func GetPublisher(ctx *fiber.Ctx) error {
-	var publisher models.Publisher
-
-	if err := db.Database.First(&publisher, ctx.Params("id")).Error; err != nil {
-		if errors.Is(err, gorm.ErrRecordNotFound) {
-			return ctx.Status(fiber.StatusNotFound).JSON(fiber.Map{
-				"message": err.Error(),
-			})
-		}
+	publisherEntity := models.Publisher{}
+	requestID := ctx.Params("id")
 
-		return ctx.Status(fiber.StatusInternalServerError).JSON(fiber.Map{
-			"message": err.Error(),
-		})
+	if err := db.Database.First(&publisherEntity, requestID).Error; err != nil {
```

Comment:
```suggestion
	if err := db.Database.First(&publisherEntity, ctx.Params("id")).Error; err != nil {
```

`requestID` is actually a publisherID, but we can avoid giving it a name by inlining it